components/automate-ui/src/app/components/calendar/calendar.component.ts

```diff
@@ -3,7 +3,7 @@ import { Component,
          Output,
          EventEmitter,
          ChangeDetectionStrategy } from '@angular/core';
-import * as m from 'moment';
+import * as moment from 'moment/moment';
```

Comment:
This is the workaround for `moment` mentioned in the preamble. This one particular file was coincidentally unique by starting with `import * as m from...` whereas every other file had the canonical `import * as moment from...` so this file changes to that standard, too.